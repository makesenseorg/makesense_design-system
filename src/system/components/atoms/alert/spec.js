import { shallowMount } from "@vue/test-utils";
import Comp from "./Alert.vue";

describe("Alert.vue", () => {
  it("matches snapshot", () => {
    const wrapper = shallowMount(Comp);
    expect(wrapper.element).toMatchSnapshot();
  });

  it("sets class with type modifier", () => {
    const wrapper = shallowMount(Comp, {
      propsData: {
        type: "positive",
      },
    });
    expect(wrapper.classes()).toContain("alert--positive");
  });
});
