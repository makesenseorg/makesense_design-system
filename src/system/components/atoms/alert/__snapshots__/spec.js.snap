// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert.vue matches snapshot 1`] = `
<div
  class="alert alert--warning"
/>
`;
